pkg/indexmanagement/reconcile.go

```diff
@@ -79,7 +81,7 @@ func RemoveCronJobsForMappings(apiclient client.Client, cluster *apis.Elasticsea
 		client.MatchingLabels(imLabels),
 	}
 	if err := apiclient.List(context.TODO(), cronList, listOpts...); err != nil {
-		return err
+		return errors.Wrap(err, "failed to list cron jobs", "namespace", cluster.Namespace, "labels", imLabels)
```

Comment:
these should also match a certain cluster, right? can we add the cluster name to the error?